functions/index.js

```diff
@@ -143,3 +143,183 @@ app.post('/', isAuthenticated ,function(req,response)
 });
 
 exports.signUp = functions.https.onRequest(app);
+
+
+
+
+
+const db = admin.database().ref();
+
+exports.addEvent = functions.https.onRequest((req,res) => {
+
+    let eventCategory = req.body.eventCategory;
+    let eventName = req.body.eventName;
+    let startTime = req.body.startTime;
+    let endTime = req.body.endTime;
+    let eventDescription = req.body.eventDescription;
+
```

Comment:
Make generic body design of event data. Many new fields can be introduced later on. So accept form data as json and parse it here.

Reply:
okkk